### PR TITLE
Add option to change pen size with the scroll wheel

### PR DESCRIPTION
**Pull request by finnboeger**
When scrolling while holding the `alt` key the pen size will be increment or decremented.
When also holding the `shift` key, the change in size will be smaller to allow for finetuning.

### CODE REVIEW

File: client-data/tools/zoom/zoom.js [hunk 1]
```diff
@@ -82,14 +82,22 @@
...
             var scale = Tools.getScale();
             var x = evt.pageX / scale;
             var y = evt.pageY / scale;
             setOrigin(x, y, evt, false);
             animate((1 - ((evt.deltaY > 0) - (evt.deltaY < 0)) * 0.25) * Tools.getScale());
+        } else if (evt.altKey) {
+            // make finer changes if shift is being held
+            var change = evt.shiftKey ? 1 : 5;
+            // change tool size
+            Tools.setSize(Tools.getSize() - ((evt.deltaY > 0) - (evt.deltaY < 0)) * change);
```
Comment (lovasoa) [inline client-data/tools/zoom/zoom.js:95]: Couldn't we make it directly proportional to the deltaY ?

Comment (finnboeger) [reply client-data/tools/zoom/zoom.js:95]: That would be different in each browser (same issue that the scroll to zoom function had initially). Firefox reports a delta of 3 whereas chrome reports 53 (at least on my device).

**Comment by lovasoa** [comment]
I updated the code to give a visual feedback when changing the cursor size. Let me know what you think.

**Comment by finnboeger** [comment]
lgtm :+1:

**Pull request merged**